bunicorn.js.org

ragokan opened the pull request:
- [x] There is reasonable content on the page (see: [Content](https://bunicorn.vercel.app/))
- [x] I have read and accepted the [Terms and Conditions](http://js.org/terms.html)

Bunicorn is just another JS library for creating servers like express/fastify, but with more Edge support and faster runtime.

MattIPv4 reviewed (commented):
👋 Do you have a link to where we can preview your content prior to merging this PR?

ragokan commented:
> 👋 Do you have a link to where we can preview your content prior to merging this PR?

Of course, I added it to 'Content' of the description as link.
You can also see it here: https://bunicorn.vercel.app/

indus commented:
I've already added your requested subdomain to JS.ORGs zonefile. You may have to configure your host to accept the custom domain (currently it also redirects to `www.`)
I'll merge after we have "contact". Please keep this PR open.

ragokan commented:
> I've already added your requested subdomain to JS.ORGs zonefile. You may have to configure your host to accept the custom domain (currently it also redirects to `www.`) I'll merge after we have "contact". Please keep this PR open.

Hello, thank you so much for adding. Looks like it accepted the non www one now, I am not sure what should I do for www, this is my Vercel page, but if still it does not allow www, I can do redirect. Thanks!

<img width="907" alt="SS 2023-10-24 at 18 43 40" src="https://github.com/js-org/js.org/assets/63511519/2bce5806-1543-4db0-a31f-b7507a1a5f54">
<img width="907" alt="SS 2023-10-24 at 18 43 48" src="https://github.com/js-org/js.org/assets/63511519/e9d7b803-70c3-469f-a145-06a29a77ff86">

ragokan commented:
@indus Since I had some problems with setting it on Vercel, and your docs shows exactly how to do for 'Github Pages', I moved it to Github Pages. Now, the new url is: https://ragokan.com/bunicorn/

I also updated the records on JS file. Thanks for helping!

MattIPv4 reviewed (changes requested):
Hi there!

❌ Before we can accept your domain request, you need to configure your GitHub Pages site to use the requested js.org subdomain. Head to your repository settings, locate the Pages settings (under Code and Automation), and enter the js.org subdomain in the custom domain field. 📝

ragokan commented:
> Hi there!
> 
> ❌ Before we can accept your domain request, you need to configure your GitHub Pages site to use the requested js.org subdomain. Head to your repository settings, locate the Pages settings (under Code and Automation), and enter the js.org subdomain in the custom domain field. 📝

Hello, sorry for the confusions, I added it to Pages settings now, domain check is successful, too.

indus commented:
@ragokan now I get a redirect error. There is something really messed up.
Because you are using a workflow to deploy the page, additional steps may be required to make the CNAME work.
FYI: https://github.com/js-org/js.org/pull/7853#issuecomment-1336537858 / https://github.com/js-org/js.org/pull/8532#issuecomment-1732508431

ragokan commented:
> @ragokan now I get a redirect error. There is something really messed up. Because you are using a workflow to deploy the page, additional steps may be required to 
make the CNAME work. FYI: [#7853 (comment)](https://github.com/js-org/js.org/pull/7853#issuecomment-1336537858) / [#8532 (comment)](https://github.com/js-org/js.org/pull/8532#issuecomment-1732508431)

Thanks, I'll search for them.

This is current settings:
<img width="907" alt="SS 2023-10-25 at 18 32 02" src="https://github.com/js-org/js.org/assets/63511519/327d4431-57c8-42a0-a06d-3cdf4fcf2ffb">

indus commented:
Welcome https://bunicorn.js.org
